docs/specification/README.md

```diff
@@ -28,6 +28,7 @@ MCAP files are designed to work well under various workloads, resource constrain
   - [Metadata Index](#metadata-op0x0C)
   - [Summary Offset](#summary-offset-op0x0D)
   - [Data End](#data-end-op0x0E)
+  - [Schema](#schema-op0x0F)
```

Comment:
move up and reorder opcodes?

Reply:
yea - we will reorder the opcodes once we are happy with the records